includes/Traits/Find_Readme.php

```diff
@@ -32,7 +32,13 @@ protected function filter_files_for_readme( array $files, $plugin_relative_path
 			$readme_list,
 			function ( $file ) use ( $plugin_relative_path ) {
 				$file = str_replace( $plugin_relative_path, '', $file );
-				return ! str_contains( $file, '/' );
+
+				$info = pathinfo( $file );
```

Comment:
Can't we just fix the regex above?

```diff
-$readme_list = preg_grep( '/readme\.(txt|md)$/i', $files );
+$readme_list = preg_grep( '/\/readme\.(txt|md)$/i', $files );
```

A test could help verify this.

Reply:
Ok, I will check.

Reply:
Updated and test added.